lib/keycloak/volumes/secrets.env

```diff
@@ -0,0 +1,3 @@
+KEYCLOAK_USER=kcadmin
+KEYCLOAK_PASSWORD=admin
+PROXY_ADDRESS_FORWARDING=false
```

Comment:
It is recommended to store `env` values inside `.env` in the project root directory. Please see `etc/env/example.env`